src/simsopt/configs/zoo.py

```diff
@@ -2,11 +2,19 @@
 from simsopt.geo.curverzfourier import CurveRZFourier
 from simsopt.geo.curvexyzfourier import CurveXYZFourier
 from simsopt.field.coil import Current
+from simsopt._core.dev import SimsoptRequires
+from simsopt._core.json import GSONDecoder
+import json
+
+try:
+    import requests
```

Comment:
Is "requests" already installed in the CI, or does it need to be added so the test will run?